### DISCUSSION TITLE
Invoicing inquiry, None Japanese tax issue

### DISCUSSION THREAD

**Discussion by kawamra** [community/community#121897] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### Body

May I ask your favor?
from our company finance division and auditor,
We wish we need "Japanese tax 10% included" written for Invoice.

How we connecting for Github finance division?

I sent inquiry form : Habe been done. but, None acknowledgement.

**Comment by ebndev** [comment, marked as answer]
Hi @kawamra,

This community is managed by GitHub employees, but it is not a support forum. The only place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. 

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!
